### DISCUSSION TITLE
Suggesting the Removal of the "Copy toot"

### DISCUSSION THREAD

**Discussion by sevi418** [tooot-app/app#713] [General]
While this feature may seem helpful to some users, it may unintentionally infringe on users' privacy and control over their content.

The ability to copy another user's post content without their explicit permission could potentially lead to misuse, such as reposting the content without proper attribution or using it for malicious purposes. Furthermore, this feature could facilitate the spread of misinformation and hate speech by making it easier for users to copy and paste inflammatory content.

---
Or just allow user to copy their own's post/toot?

**Comment by xmflsct** [comment]
I understand your concern. But it is always a balance of choice. There is no way to truely prevent content being copied, at a bottom line.

Content being redistributed is not the most dangerous part, everyone can share any content, even not copied, as misinformation. The most threatening part is one being linked to what they have posted, outside of its original context. Hence tooot disallow screenshots being taken if on Android, and warn users of doing so on iOS as iOS does not offer the possibility to fully disallow taking screenshots.

Currently "copy toot" is only shown when a toot is opened. The barrier is high enough before anyone can copy the content.

**Comment by sevi418** [comment]
But the copy toot option is just below the share toot option, it just implies to the user that _instead of sharing it with urls you could just copy it without concern._
